### ISSUE TITLE
[Feature] Support on disk live boot entry for real dual boot

### ISSUE THREAD

**Issue by freechelmi** [hakuna-m/wubiuefi#21]
this is a pretty big feature : 

- Instead of installing Ubuntu on virtualDisk file in NTFS partition, add a Windows boot manager entry that boot the liveCD to RAM with the preseed file .

- Live CD will unmount the windows partition to resize it as chosen an install as usual.

- Aready done by debian team  : http://goodbye-microsoft.com/

**Comment by hakuna-m** [comment]
> * Instead of installing Ubuntu on virtualDisk file in NTFS partition, add a Windows boot manager entry that boot the liveCD to RAM with the preseed file .

Theoretically, this feature still exists. see [CD boot helper](https://wiki.ubuntu.com/Testing/Cases/Wubi)

> In this mode Wubi adds a boot menu entry to allow users to boot off the CD without having to change the device boot order in the bios. 

There is also an option **--cdboot** to run this feature. e.g.

`wubi.exe --cdboot`

But IMHO it is necessary to maintain it to get it working. I add it to the task list.

**Comment by freechelmi** [comment]
Yes CdBoot is a great featire but what I propose here is totally different.

- User wants to dual boot linux.

- All he has to do is download & run the small Wubi exe & answer few questions

- He never has to burn CD , understand what is an iso image etc....

**Comment by hakuna-m** [comment]
> but what I propose here is totally different.

I think I know what you mean. An installation like "Ubuntu alongside Windows" of the standard installer 
but with all features of Wubi for an easy, safe and automatic installation.

The problem is that some Wubi features depends on the installation method. e.g. The feature that you can select a Windows drive or the feature that an uninstaller removes all changes of the installation.

Of course, it is also possible to change the uninstaller that it deletes Ubuntu partitions and resizes Windows partitions. But is it really as safe for Windows as current Wubi uninstaller? Same question if we change installer that it resizes partitions.

On my experience, ["Goodbye Microsoft"](http://goodbye-microsoft.com) as the name of the debian project site says, is not really the most important feature for a lot of Wubi users.

So IMHO, there are only few users who would use the new features with Wubi. So it makes not really sense to develop it.

The standard installer Ubiquity is also easy to use and it can also install Ubuntu alongside Windows. So, I suggest to add some features to the existing boot helper. Of course, it is not enough to be a boot helper for an existing and burned DVD/CD....

**Comment by freechelmi** [comment]
Hi , going back to this feature as a lot a remote windows user ask for help to create dualBoot.

- I totally understand your points and thank you for the details.
- I will try a mockup but the uninstaller will never be implemented (although "safe" from Windows) 
- You are right on the point that resising ( from linux live installer ) the windows partition is always a risk, there will be a disclaimer for backups.
- Yes you can just see this feature as a new "iso Boot helper"

**Comment by freechelmi** [comment]
I 've changed the title and description as my first idea was really messy.

I've wrote a small powershell script that resize the C: Windows sucessfully, 

I'm trying now on a branch to tell "Wubi" to install on available space instead of NTFS files.

Yes for sure there will not be an "uninstaller" at first but Linux is here to stay :-) 

Thanks for your work on Wubi !

**Comment by freechelmi** [comment]
Hi just to say we started a new project to fullfil this feature : https://libertix.eu
